dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -143,4 +143,6 @@ RUN azure-cli_bundle_*/installer
 # TODO: if azure-cli contains newer version azure-keyvault, remove this
 RUN ~/lib/azure-cli/bin/python -m pip install azure-keyvault==0.3.7 -U
 
+RUN apt-get install -y python-subnettree
```

Comment:
can you use pip to install since pip have more latest version.

Reply:
Please help to review, thanks a lot.